### PR TITLE
Bugfix/vmo 5873/unable to open block editors

### PR DESCRIPTION
**Pull request by bulatgab**

### CODE REVIEW

File: src/common-imports.ts [hunk 1]
```diff
@@ -26,9 +27,11 @@ export default function registerCustomComponents(extra = {}): void {
...
     ...resourceEditorComponents,
     ...extra,
-  })
+  }
+  Object.entries(Components)
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     .forEach(([name, component]: [string, any]) => {
       Vue.component(name, component)
     })
+  return Components
```
Comment (bulatgab) [inline src/common-imports.ts:36]: These changes in `src/common-imports.ts` are unrelated to the bug.
They fix the return from the `registerCustomComponents()` that is used in the file below for default exports (in `src/lib.ts`)

**Review by object-required — APPROVED** [review]

File: src/common-imports.ts (hunk 1 above)
Comment (safydy-r) [reply src/common-imports.ts:36]: Ok, thanks for doing that! 🥇

File: src/views/InteractionDesigner.vue [hunk 2]
```diff
@@ -214,13 +214,15 @@ export class InteractionDesigner extends mixins(Lang, Routes) {
     }
   }
 
-  beforeRouteUpdate(to: Route, from: Route, next: Function): void {
+  @Watch('$route', {deep: true})
+  handleRouteUpdate(to: Route): void {
+    console.log('Bulat this', this)
+    console.log('Bulat this.$route', this.$route)
```
Comment (object-required) [inline src/views/InteractionDesigner.vue]: I believe using `@Watch` is perfectly justified here 👍

**Review by object-required — APPROVED** [review]

File: src/views/InteractionDesigner.vue [hunk 3]
```diff
@@ -214,13 +214,13 @@ export class InteractionDesigner extends mixins(Lang, Routes) {
     }
   }
 
-  beforeRouteUpdate(to: Route, from: Route, next: Function): void {
+  @Watch('$route', {deep: true})
+  handleRouteUpdate(to: Route): void {
```
Comment (safydy-r) [inline src/views/InteractionDesigner.vue:218]: 💯 , thanks for this

**Review by yeboaasare — APPROVED** [review]

**Pull request merged**